scripts/pi-hole/php/update_checker.php

```diff
@@ -1,90 +1,54 @@
 <?php
-// Allow file_get_contents to read remote sources via URL
-ini_set("allow_url_fopen", 1);
 
-// Function that grabs latest tag from GitHub
-function get_github_version($url)
-{
-	// Have to fake the user agent to be able to query from Github's API
-	$context = stream_context_create(array("http" => array("user_agent" => "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36")));
-
-	$json = file_get_contents($url, false, $context);
-	$result = json_decode($json);
+$localversions = "/etc/pihole/localversions";
+$localbranches = "/etc/pihole/localbranches";
+$GitHubVersions = "/etc/pihole/GitHubVersions";
 
-	return $result->tag_name;
+if(!is_readable($localversions) || !is_readable($localbranches) || !is_readable($GitHubVersions))
+{
+	$core_branch = "master";
+	$core_current = "N/A";
+	$core_update = false;
+	$web_branch = "master";
+	$web_current = "N/A";
+	$web_update = false;
+	$FTL_current = "N/A";
+	$FTL_update = false;
+	goto endofupdatecheck;
```

Comment:
We really shouldn't use `goto`. How about move this into a function and return at this point, or exit if you can do that and not stop the rest of our PHP code.